### PR TITLE
allow to run prometheus on a remote node

### PR DESCRIPTION
**Pull request by fixxxedpoint**
allows to run prometheus on a remote node instead of on a local machine

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -478,3 +477,26 @@ def fab_cmd():
...
     with open('node_flags.json', 'w') as f:
         json.dump(flags, f)
+
+
+def convert_to_targets(ips: List[str]) -> List[str]:
+    return [f'{ip}:9615' for ip in ips]
+
+
+def convert_to_node_targets(ips: List[str]) -> List[str]:
+    return [f'{ip}:9100' for ip in ips]
```
Comment (kostekIV) [inline utils.py:487]: nit: just one function `def convert_to_targets(ips: List[str], port: int) ...`

File: shell.py [hunk 2]
```diff
@@ -615,3 +616,30 @@ def run_devnet(n_parties, regions=use_regions(), instance_type='t2.micro'):
     run_task('run-docker-compose', regions, parallel, False, pids)
 
     instances_state(testnet_regions(), 'testnet')
+
+
+def setup_prometheus(region=default_region(), tag='prometheus', target_region=default_region(), target_tag="dev"):
```
Comment (kostekIV) [inline shell.py:621]: target_region should be `target_regions` to allow nodes in multiple regions.

File: shell.py [hunk 3]
```diff
@@ -615,3 +616,30 @@ def run_devnet(n_parties, regions=use_regions(), instance_type='t2.micro'):
...
+    launch_new_instances_in_region(n_parties=1, region_name=region, tag=tag)
+    print('prometheus ip:', instances_ip_in_region(region_name=region, tag=tag))
+
+    color_print('waiting till ports are open on machines')
+    wait('open 22', [region], tag)
+
+    color_print('setup')
+    run_task('setup', regions=[region], parallel=False, tag=tag)
+
+    print('creating prometheus.yml configuration file')
```
Comment (kostekIV) [inline shell.py:636]: nit: `color_print`

**Review by kostekIV — APPROVED** [review]

**Review by mike1729 — APPROVED** [review]

File: utils.py (hunk 1 above)
Comment (fixxxedpoint) [reply utils.py]: changed

File: shell.py (hunk 2 above)
Comment (fixxxedpoint) [reply shell.py]: changed

File: shell.py (hunk 3 above)
Comment (fixxxedpoint) [reply shell.py]: changed

**Pull request merged**